### ISSUE TITLE
LoadError: cannot load such file -- coveralls

### ISSUE THREAD

**Issue by robcole** [AuthorizeNet/sdk-ruby#83]
Version: 1.8.8

After updating our project to version 1.8.8 of the gem, we receive the following error:

```
==> default: LoadError: cannot load such file -- coveralls
==> default: /home/vagrant/useed/config/application.rb:14:in `<top (required)>'
==> default: /home/vagrant/useed/Rakefile:5:in `require'
==> default: /home/vagrant/useed/Rakefile:5:in `<top (required)>'
==> default: (See full trace by running task with --trace)
```

Seems related to https://github.com/AuthorizeNet/sdk-ruby/commit/a7630fb6e68174ef9697eef3897adcf911166653 which looks like it should only be called in the `spec_helper.rb` file.

**Comment by namanbansal** [comment]
Hi @robcole ,

Thanks for reporting. We have released a bumped up version 1.8.8.1 of the gem, which should fix the issue.
Please let us know if you still face any issues.

**Comment by brianmc** [comment]
Thanks Naman.
